configs/dictionaries/ab_test_expiries.yaml

```diff
@@ -8,3 +8,4 @@
 Example: 86400
 # 91 days and 6 hours (~ 3 months)
 AccountBankHols: 7884000
+ExploreMenuAbTestable: 1209600
```

Comment:
if you're starting the AB test with 100 in A and then wanting to switch some people over, then having a long time here will make it harder to increase your B cohort quickly because they might all be stuck in A until their cookie expires. Could be worth starting with a shorter time, and then bumping it up once you have greater confidence? 

(It's up to you though, this isn't a problem as long as you're aware)

Reply:
Double checked with Tara and it's 1 day then 3 months. Updated in